sale_commission/i18n/es.po

```diff
@@ -74,11 +81,22 @@ msgid "Agents"
 msgstr "Agentes"
 
 #. module: sale_commission
-#: field:account.invoice.line,agents:0 field:sale.order.line,agents:0
+#: field:account.invoice.line,agents:0
+#: field:sale.order.line,agents:0
 msgid "Agents & commissions"
 msgstr "Agentes y comisiones"
 
 #. module: sale_commission
+#: view:res.partner:sale_commission.view_partner_form_agent
+msgid "Agents / Commissions"
+msgstr "Agentes / Comisiones"
+
+#. module: sale_commission
+#: view:res.partner:sale_commission.view_partner_form_agent
+msgid "Agents and Commissions"
```

Comment:
Why don't you use the same spelling in all places (`Agents & commissions`) and thus there's no similar strings to translate?